releasenotes/notes/network-path-remove-noisy-log-918ab3c6d760dc5e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Removes a log statement which was causing a lot of noise in the Network Path logs.
```

Comment:
```suggestion
    Removes a log statement that was causing a lot of noise in the Network Path logs.
```

Reply:
This is a backport for 7.58, so this is already going in for 7.59. I'm thinking I'll just leave it so it's consistent rather than have it be more correct in one release and less in the other, but if you have a strong opinion let me know.